process/incubation.md

```diff
@@ -105,6 +105,9 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
   * Follow Swift Code style guidelines
   * Uses force unwraps and force tries only as preconditions, ie. conditions that the programmer regards as impossible or programmer error. All force tries/unwraps should come with a comment stating the reasons
   * Prefer native Swift over C wrapping, where appropriate
+  * Does not use `*Unsafe*` unless interfacing with C
+    * Exceptions to uses of `*Unsafe*` constructs are tolerated if appropriately documented why they are absolutely necessary
+    * When `*Unsafe*` is used in this manner, it would be appreciated for feature requests to be made for Swift or SwiftNIO in order to make the use of the `*Unsafe*` construct obsolete
```

Comment:
consider rephrasing "it would be appreciated for feature requests to be made for Swift or SwiftNIO" -> "its is expected to be accompanied by enhancement requests ticket describing the root cause in Swift or an offending library". i dont think we need to call out SwiftNIO here

Reply:
@tomerd well, without SwiftNIO you can't parse for example an integer from a buffer without "Unsafe" so there are plenty of situations where you have to. With SwiftNIO I think we have the goal to plug almost all of those holes and that's why I've specifically said SwiftNIO here.

Reply:
@weissi i get it, but would be good to make sure its clear SwiftNIO is an optional component here. or iow not all projects even use it. maybe something like: "its is expected to be accompanied by enhancement requests ticket describing the root cause in Swift, SwiftNIO, or a different offending library"

Reply:
thanks, updated